testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -437,23 +436,22 @@ def test_triclinic_box(self, lengths, angles):
                              comb_wr([-10, 0, 20, 70, 90, 120, 180], 3))
     def test_box_volume(self, lengths, angles):
         box = np.array(lengths + angles, dtype=np.float32)
-        assert_almost_equal(mdamath.box_volume(box),
-                            np.linalg.det(self.ref_trivecs(box)),
-                            decimal=5)
+        approx_det = approx(np.linalg.det(self.ref_trivecs(box)), abs=1e-5)
```

Comment:
These tests are passing but it turns out one of the values weren't scalars. Will use `assert_allclose` instead.